### ISSUE TITLE
Validation issues with OpenxQuest_example_added_annotations.mzid

### ISSUE THREAD

**Issue by edeutsch** [HUPO-PSI/mzIdentML#57]
Error at file /net/db/projects/PSI/mzIdentML/1.2/genDoc/OpenxQuest_example_added_annotations.mzid, line 7, char 37
  Message: Schema in mzIdentML1.2.0.xsd has a different target namespace from the one specified in the instance document http://psidev.info/psi/pi/mzIdentML/1.2.0.

Error at file /net/db/projects/PSI/mzIdentML/1.2/genDoc/OpenxQuest_example_added_annotations.mzid, line 204, char 81
  Message: Attribute 'values' is not declared for element 'userParam'

Error at file /net/db/projects/PSI/mzIdentML/1.2/genDoc/OpenxQuest_example_added_annotations.mzid, line 446, char 13
  Message: The key for identity constraint of element 'MzIdentML' is not found.

and many further instances of these same errors.

**Comment by mwalzer** [comment]
Fixed these errors including the ones also in the plain QXQ example, however:
@germa I get one error which I cannot make sense of:
```
Message 1:
    Level: ERROR
    --> Non-fatal XML Parsing error detected on line 527
  Error message: Key 'FK_DATAADSILSIRSIIFRAGIONFRAGARR' with value 'Measure_Int' not found for identity constraint of element 'MzIdentML'.
```
so it is actually symmetrical to the QXQ example without annotations, ```<Fragmentation>```elements  added

**Comment by mwalzer** [comment]
fixed in PR #66 update

**Comment by edeutsch** [comment]
Status today:
ERROR: cvParam retention time has an illegal unit second!
WARNING: XL:00002 should be 'cross-linker related PTM' instead of 'Xlink:DSS'
(see bug report for other OXQ file for more details on what the real problem is here)

**Comment by mwalzer** [comment]
I think these relate to files prior to https://github.com/HUPO-PSI/mzIdentML/pull/66

**Comment by mwalzer** [comment]
I think these relate to files prior to PR https://github.com/HUPO-PSI/mzIdentML/pull/66 ?

Easy access to the files from the PR here:
https://github.com/HUPO-PSI/mzIdentML/issues/52#issuecomment-230726074

**Comment by germa** [comment]
@mwalzer: Use of XLMOD:nnnnn instead of XL:nnnnn, see also GitHub issue #60

**Comment by edeutsch** [comment]
Today's validation results:
WARNING: CV term XL:00002 ('DSS') is not in the cv

This needs to be XLMOD: as pointed out above.

**Comment by mwalzer** [comment]
fixed in #77
